src/headless/celery_app.py

```diff
@@ -13,11 +18,14 @@
     'headless',
 )
 
-
-# initialize qgis_app
+# Initialize qgis_app
 from safe.test.utilities import get_qgis_app  # noqa
 QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
 
+if OUTPUT_DIRECTORY:
+    if not os.path.exists(OUTPUT_DIRECTORY):
```

Comment:
Please, change this into try except OSError.

Reply:
For reference: https://stackoverflow.com/questions/273192/how-can-i-create-a-directory-if-it-does-not-exist

Reply:
Done.